components/eventing-controller/config/crd/bases/eventing.kyma-project.io_subscriptions.yaml

```diff
@@ -0,0 +1,55 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Is it possible to generate code for "apiextensions.k8s.io/v1" cause "v1beta1" is already deprecated.

Reply:
True, I fixed it.